### PR TITLE
Feature/setup test env

### PR DESCRIPTION
**Pull request by kshaaban-**
- Finalize test environment setup
- Write unit tests for SignupPage

Also:
- Add `.travis.yml` config file for CI
  I requested that we grant TravisCI access to the repo
- Add [coveralls](https://coveralls.io/) for test coverage reporting
  Also requested that we gran coveralls access to the repo

### CODE REVIEW

**Comment by kshaaban-** [comment]
This should close #42

File: .angular-cli.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      "index": "index.html",
+      "main": "app/main.ts",
+      "polyfills": "../test/polyfills.ts",
+      "test": "../test/test.ts",
+      "tsconfig": "../test/tsconfig.app.json",
+      "testTsconfig": "../test/tsconfig.spec.json",
+      "prefix": "app",
+      "scripts": [],
+      "environmentSource": "environments/environment.ts",
+      "environments": {
```
Comment (yannbf) [inline .angular-cli.json]: Really nice to already think about the environments and set this up! 👍

File: .angular-cli.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "project": {
+    "name": "my-ng-app"
```
Comment (yannbf) [inline .angular-cli.json]: I think this should be bitfindr, right?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # Bitfindr
 [![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors)
-[![Gitter chat](https://img.shields.io/badge/chat-on_gitter-008080.svg?style=flat-square)](https://gitter.im/bitfindr)
+[![Build Status](https://travis-ci.org/bitfindr/bitfindr.svg?branch=master)](https://travis-ci.org/bitfindr/bitfindr)
```
Comment (yannbf) [inline README.md]: Thanks for already adding the badges :DD

File: .angular-cli.json (hunk 2 above)
Comment (kshaaban-) [reply .angular-cli.json]: Yeah I think so :D

**Pull request merged**